packages/base-types/src/models/project.ts

```diff
@@ -23,13 +23,12 @@ export interface ProjectPrototypeLuis extends ProjectPrototypeNLPBase {
 export type ProjectPrototypeNLP = ProjectPrototypeLuis;
 
 export interface ProjectPrototype {
-  data: Record<string, any>;
+  data?: Record<string, any>;
```

Comment:
i dont know where this is being used, couldnt find any usages on creator, switched it to optional cause adding the messageDelay stuff was complaining it didnt have this data prop. not sure what its used for...